client/webserver/site/src/js/markets.js

```diff
@@ -959,6 +959,16 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (note) {
     app.log('book', 'handleBookRoute:', note)
     const mktBook = note.payload
+    const { book } = mktBook
+    // gets first price value from buy or from sell, so we can show it on
+    // title.
+    const firstKnownValue = book.buys[0] ? book.buys[0] : book.sells[0]
+    if (firstKnownValue) {
+      // remove rate from title if already have it.
+      document.title = (document.title).replace(/\d+([.])?/g, '')
+      // more than 6 numbers it gets too big for the title.
+      document.title = `${firstKnownValue.rate.toFixed(6)} ${document.title}`
```

Comment:
I'd like to see the market name. Something like `0.00512 DCR/BTC`.

I don't know that we need to save the original title, which said `Markets | Decred DEX`, but if you do want to append the original text, grab a copy in the `MarketPage` constructor, `this.ogTitle = document.title` and just append it rather than using `replace`.

Reply:
I removed the replace and now I am adding the selected assets into the title. 

Added a new method for updating it, and now I call it on `handleBookRoute`, `handleUnbookOrderRoute`, and `handleBookOrderRoute`.

This way if we adding or removing new bids, it will update the title based on it.

Reply:
> I'd like to see the market name. Something like `0.00512 DCR/BTC`.

Despite calling it the market name, is giving units to that rate, so it'll need to be BTC/DCR

Reply:
Discussion in this thread: https://github.com/decred/dcrdex/pull/1117#discussion_r671288493
Looks like use "DCR-BTC" and don't have the number with units.